docs/manage/security/rbac.rst

```diff
@@ -475,6 +475,12 @@ edit, or delete projects and experiments within its scope.
 The ``Editor`` role supersedes the ``EditorRestricted`` role and includes permissions to create or
 update NTSC tasks within its scope.
 
+- ``EditorRestricted`` users can still open and use scoped JupyterLab notebooks and perform all
+experiment-related jobs as those with the ``Editor`` role. The only additional permissions granted
+by the ``Editor`` role are the permission to create notebooks, tensorboards, shells, and commands
+(NTSC tasks) and the permssion to perform updates on those tasks (such as changing the task's
+priority or deleting it).
+
```

Comment:
- ``EditorRestricted`` users can still open and use scoped JupyterLab notebooks and perform all
experiment-related jobs, just like those with the ``Editor`` role. The only additional permissions granted
by the ``Editor`` role include the ability to create notebooks, TensorBoards, shells, and commands
(NTSC tasks), as well as the permission to update these tasks, such as changing the task's priority or deleting it.